server.go

```diff
@@ -130,6 +130,21 @@ func runMain() error {
 			err)
 	}
 
+	stakepooldColdWalletResponses, err := controller.Cfg.StakepooldServers.GetAllColdWalletExtPub()
+	if err != nil {
+		log.Infof("RPC GetColdWalletExtPub failed: %v", err)
+		return fmt.Errorf("GetColdWalletExtPub: RPC server error %v", err)
+	}
+
+	// Check that coldwalletextpub is the same in dcrstakepool.conf and all stakepoold.conf files
+	for _, stakepooldColdWalletResponse := range stakepooldColdWalletResponses {
+		if cfg.ColdWalletExtPub != stakepooldColdWalletResponse.ColdWalletExtPub {
+			err := fmt.Errorf("coldwalletextpub is not the same in dcrstakepool.conf and stakepoold.conf files")
+			fmt.Fprintln(os.Stderr, err)
+			return err
+		}
+	}
+
```

Comment:
Can you combine both of these into one method on stakepooldclient that takes the xpub and gives back ok or not?

Reply:
agree